integration-tests/integration_test.go

```diff
@@ -802,3 +802,81 @@ func (suite *ApexIntegrationSuite) TestRelayNAT() {
 	require.NoError(err)
 	assert.Equal(5, lc, "the number of expected wg show peers was %d, found %d: wg show out: \n%s", 5, lc, wgSpokeShow)
 }
+
+func (suite *ApexIntegrationSuite) TestApexCtl() {
+	assert := suite.Assert()
+	require := suite.Require()
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, 60*time.Second)
+	defer cancel()
+
+	token, err := getToken(ctx, "kitteh5@apex.local", "floofykittens")
+	require.NoError(err)
+
+	// create the nodes
+	node1 := suite.CreateNode(ctx, "node1", []string{defaultNetwork})
+	suite.T().Cleanup(func() {
+		if err := node1.Terminate(parentCtx); err != nil {
+			suite.logger.Errorf("failed to terminate container %v", err)
+		}
+	})
+	node2 := suite.CreateNode(ctx, "node2", []string{defaultNetwork})
+	suite.T().Cleanup(func() {
+		if err := node2.Terminate(parentCtx); err != nil {
+			suite.logger.Errorf("failed to terminate container %v", err)
+		}
+	})
+
+	c, err := newClient(ctx, token)
+	require.NoError(err)
+
+	// create a new zone
+	zoneID, err := c.CreateZone("apexctl-zone",
+		"zone full of apexctls",
+		"172.19.100.0/24",
+		false)
+	require.NoError(err)
+
+	// TODO: debug the non-utf8 character coming back from suite.containerExec --> �{
+	//zoneOut, err := suite.containerExec(ctx, node1,
+	//	[]string{"/bin/apexctl",
+	//		"--username", "kitteh5@apex.local",
+	//		"--password", "floofykittens",
+	//		"--output", "json",
+	//		"zone", "create",
+	//		"--name", "zone-apexctl",
+	//		"--cidr", "172.19.100.0/24",
+	//		"--description", "apexctl e2e zone",
+	//	})
+	//assert.NoErrorf(err, "apexctl zone create error: %v\n", err)
+	//var zone models.Zone
+	//err = json.Unmarshal([]byte(zoneOut), &zone)
+
+	moveOut, err := suite.containerExec(ctx, node1,
+		[]string{"/bin/apexctl",
+			"--username", "kitteh5@apex.local",
+			"--password", "floofykittens",
+			"zone", "move-user",
+			"--zone-id", zoneID.ID.String(),
+		})
+	assert.NoErrorf(err, "apexctl zone move-user error: %v\n", err)
+	suite.logger.Infof("move out: %s\n", moveOut)
+
+	// start apex on the nodes
+	go suite.runApex(ctx, node1, fmt.Sprintf("--with-token=%s", token))
```

Comment:
Do we even need to start the nodes? Because we only need to create the zone to verify that apexctl is working as expected.

Reply:
Firing up nodes as part of IPAM pool testing, when a device gets deleted that node's addresses need to be returned to the pool. This is part of validating that. Same goes for Zone deletion/prefix in this last push.